cmd/chainbridge/flags.go

```diff
@@ -56,11 +56,6 @@ var (
 
 // Deploy Contracts Flags
 var (
-	PortFlag = cli.StringFlag{
-		Name:  "port",
-		Usage: "The port at which your local chain instance or ganache running",
-		Value: "8545",
-	}
 	NumRelayersFlag = cli.IntFlag{
 		Name:  "validators",
 		Usage: "Specify total number of validators",
```

Comment:
I'm pretty sure all of these flags should be removed under `// Deploy contracts flags`